### PR TITLE
DEVDOCS-2076: Adding JavaScript

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2076](https://jira.bigcommerce.com/browse/DEVDOCS-2076)

## What changed?
* Applied style guide updates.

### CODE REVIEW

File: docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md [hunk 1]
```diff
@@ -1,74 +1,57 @@
...
----
-
-<a href='#adding_development-options' aria-hidden='true' class='block-anchor'  id='adding_development-options'><i aria-hidden='true' class='linkify icon'></i></a>
+Many third-party JavaScript components are distributed with npm (Node Package Manager). When you use the npm command-line utility to add a JavaScript component to your theme, Stencil will automatically bundle and minify the component. To enable this bundling and minification, run each module's `npm install` command from the root directory of your theme.
 
-## Page Types and Javascript API
+### Taking over from npm
 
-<a href='#adding_page-types' aria-hidden='true' class='block-anchor'  id='page-types_stencil-utils'><i aria-hidden='true' class='linkify icon'></i></a>
+npm facilitates managing third-party JavaScript components by placing each JavaScript component in the correct directories. However, as a developer, you will still need to edit your theme files to wire up the JavaScript component to expose it on your storefront. You will find several examples of this on the following pages.
```
Comment (uwikaiddi) [inline docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md:50]:
```suggestion
The npm package manager facilitates the management of third-party JavaScript components by placing each JavaScript component in the correct directories. However, as a developer, you will still need to edit your theme files to wire up the JavaScript component to expose it on your storefront. You will find several examples of this on the following pages.
```

File: docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md [hunk 2]
```diff
@@ -338,88 +259,45 @@ export default class Cart extends PageManager {
...
+<body>
+<script>window.__webpack_public_path__ = "{{cdn 'assets/dist/'}}";</script>
+<script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
+<script>window.stencilBootstrap("{{page_type}}", {{jsContext}}).load();</script>
 ```
 
-In order to successully map your custom module to a custom template file, that file must
-* Inject the custom template 
-* Load webpack 
+In order to successully map your custom module to a custom template file, that file must do the following:
```
Comment (uwikaiddi) [inline docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md:291]: Do the following need to be done in this specific order? If so, this should be an ordered list.

File: docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md [hunk 3]
```diff
@@ -1,74 +1,57 @@
...
+* **Place modules in assets/js/:** For a JavaScript module that is not distributed via npm, add this module to your theme by creating a subdirectory within assets/js/ that contains your module.
+* **Theme-specific JavaScript modules:** Stencil themes include their own custom JavaScript modules for most page types. You can alter these page-type–specific modules by editing the files in assets/js/theme/*.js.
 
 These techniques are outlined in the following sections.
 
----
-
-<a href='#adding_using-npm' aria-hidden='true' class='block-anchor'  id='adding_using-npm'><i aria-hidden='true' class='linkify icon'></i></a>
-
 ## Using npm (Node Package Manager)
```
Comment (uwikaiddi) [inline docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md:44]:
```suggestion
## Using npm
```

File: docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md [hunk 4]
```diff
@@ -1,74 +1,57 @@
...
-
-### Taking Over from npm
-
-npm facilitates managing third-party JavaScript components by placing each JavaScript component – and any of its dependencies – in the correct directories. However, as a developer, you will still need to edit your theme files to wire up the JavaScript component to expose it on your storefront. You will find several examples of this on the following pages.
-
----
-
-<a href='#adding_development-options' aria-hidden='true' class='block-anchor'  id='adding_development-options'><i aria-hidden='true' class='linkify icon'></i></a>
+Many third-party JavaScript components are distributed with npm (Node Package Manager). When you use the npm command-line utility to add a JavaScript component to your theme, Stencil will automatically bundle and minify the component. To enable this bundling and minification, run each module's `npm install` command from the root directory of your theme.
```
Comment (uwikaiddi) [inline docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md:46]:
```suggestion
Many third-party JavaScript components are distributed with npm. When you use the npm command-line utility to add a JavaScript component to your theme, Stencil will automatically bundle and minify the component. To enable this bundling and minification, run each module's `npm install` command from the root directory of your theme.
```

File: docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md (hunk 2 above)
Comment (aglensmith) [reply docs/stencil-docs/javascript-and-event-hooks/adding-javascript.md:291]: Nah, its just a general list of requirements.

**Pull request merged**